### PR TITLE
feat: scroll to top widget

### PR DESCRIPTION
**Pull request by Viczei**

### CODE REVIEW

File: packages/code-du-travail-frontend/src/outils/Components/SimulatorLayout.tsx [hunk 1]
```diff
@@ -125,14 +126,13 @@ const SimulatorContent = <StepName extends string>({
...
-          setNavigationAction("next");
           break;
         case ValidationResponse.Valid:
           nextStep();
-          setNavigationAction("next");
-          window?.scrollTo(0, 0);
           break;
       }
+      setNavigationAction("next");
+      scrollToTop();
```
Comment (carolineBda) [inline packages/code-du-travail-frontend/src/outils/Components/SimulatorLayout.tsx:135]: :+1:

**Review by carolineBda — APPROVED** [review]

File: packages/code-du-travail-frontend/src/outils/common/utils/scrollToTop.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function scrollToTop(): void {
+  window.scrollTo(0, 0);
+  window.parent?.postMessage({ kind: "scroll-to-top" }, "*");
+}
```
Comment (maxgfr) [inline packages/code-du-travail-frontend/src/outils/common/utils/scrollToTop.ts:4]: 👌

**Review by maxgfr — APPROVED** [review]
top (sans jeu de mot) 🙌

**Review by m-maillot — APPROVED** [review]
:+1:

**Pull request merged**